test/example.rb

```diff
@@ -2,6 +2,9 @@ def greeting(name)
   puts "Hello, #{name}! Welocome to the Ruby typos test."
   puts 'languege' # typo
```

Comment:
```suggestion
  puts 'knowlege' # typo
```
Please check this typo. Replace `languege` with `language`.